src/DOM/HTML/Window.purs

```diff
@@ -13,3 +42,71 @@ foreign import location :: forall eff. Window -> Eff (dom :: DOM | eff) Location
 foreign import innerWidth :: forall eff. Window -> Eff (dom :: DOM | eff) Int
 
 foreign import innerHeight :: forall eff. Window -> Eff (dom :: DOM | eff) Int
+
+foreign import alert :: forall eff. Window -> String -> Eff (alert :: ALERT | eff) Unit
+
+foreign import close :: forall eff. Window -> Eff (window :: WINDOW | eff) Unit
+
+foreign import closed :: forall eff. Window -> Eff (window :: WINDOW | eff) Boolean
+
+foreign import confirm :: forall eff. Window -> String -> Eff (confirm :: CONFIRM | eff) Boolean
+
+foreign import length :: forall eff. Window -> Eff (window :: WINDOW | eff) Int
+
+foreign import minimize :: forall eff. Window -> Eff (window :: WINDOW | eff) Unit
+
+foreign import moveBy :: forall eff. Window -> Int -> Int -> Eff (window :: WINDOW | eff) Unit
+
+foreign import moveTo :: forall eff. Window -> Int -> Int -> Eff (window :: WINDOW | eff) Unit
+
+open
+  :: forall eff
+   . Window
+  -> String
+  -> String
+  -> String
+  -> Eff (window :: WINDOW | eff) (Maybe Window)
+open = _open Just Nothing
+
+foreign import _open
+  :: forall eff
+   . (forall a. a -> Maybe a)
+  -> (forall a. Maybe a)
+  -> Window
+  -> String
+  -> String
+  -> String
+  -> Eff (window :: WINDOW | eff) (Maybe Window)
```

Comment:
You could implement this with `Nullable` instead and then use `toMaybe`. Something like this:

https://github.com/purescript-contrib/purescript-dom/blob/f7b786d9f005df0a41b1da63d3525b8d8ae55457/src/DOM/HTML/Event/DragEvent/DataTransfer.purs#L20-L23

The FFI code can just be currying then, with no need to handle the `Nothing` / `Just` arguments.